Packages/com.unity.inputsystem/InputSystem/Plugins/Android/InputSettingsAndroid.cs

```diff
@@ -0,0 +1,51 @@
+#if UNITY_EDITOR || UNITY_ANDROID || PACKAGE_DOCS_GENERATION
+using System;
+
+namespace UnityEngine.InputSystem
+{
+    public partial class InputSettings
+    {
+        /// <summary>
+        /// Project-wide input settings for the Android platform.
+        /// </summary>
+        [Serializable]
+        public class AndroidSettings
```

Comment:
Why would this be something Android specific when we advertise it as cross-platform on 3 operating systems for the UnityEngine.Input counterpart? https://docs.unity3d.com/ScriptReference/Input-backButtonLeavesApp.html

Reply:
`backButtonLeavesApp` is a legacy input setting. It's only living on Android land now. Even if we have the code for Windows platforms still living in our sourcecode, there are no supported device that has a physical back button anymore. They only had them in Windows Phones and early Windows RT tablets, none of which are supported anymore.

So it makes sense that moving innto the new Input System, we only added in the only device that still has a back button functionality, Android.

Reply:
@ekcoh `backButtonLeavesApp` is a legacy input setting. It's only living on Android land now, it is needed for `predictive back gesture` to work (which is something we have implemented for Android 14 as per Google's request). 

Even if we have the code for `Windows` platforms still living in unity/unity, there are no supported device that has a physical back button anymore. They only had them in Windows Phones and early Windows RT tablets, none of which are supported anymore. This has been confirmed by the windows-platform team.

So it makes sense that moving into the new Input System, we only added in the only device that still has a back button functionality, Android.